src/js/claims-status/components/ClosedClaimMessage.jsx

```diff
@@ -43,7 +43,7 @@ export default function ClosedClaimMessage({ claims, onClose }) {
         <i className="fa fa-close va-alert-close-icon" aria-label="Close icon"></i>
       </button>
       <div className="usa-alert-body">
-        <h5 className="usa-alert-heading">Recently closed:</h5>
+        <h4 className="usa-alert-heading">Recently closed:</h4>
         {closedClaims.map(claim => (
```

Comment:
This one is using the `usa-alert-heading` class, while the other ones are using `usa-alert-title`

Reply:
Good catch. I'll swap it back to `title` for now.